code/game/objects/items/grenades/grenade.dm

```diff
@@ -102,6 +102,7 @@
 
 /obj/item/grenade/proc/log_grenade(mob/user)
 	log_bomber(user, "has primed a", src, "for detonation")
+	log_combat(user, src, "has primed a", src, "for detonation")
```

Comment:
Should just have log bomber handle this for everything

Reply:
log bomber doesn't appear in individual player logs in-game, which is what I was hoping to solve with this.

Reply:
i meant add a log_combat line to the log_bomber proc so it's just automatic for everything that uses it.

Reply:
![image](https://user-images.githubusercontent.com/6209658/141045672-b88c7076-c287-46f1-92c0-74f601f5d1ab.png)
you're fixing the wrong thing in that case

Reply:
this should be going to LOG_ATTACK instead of LOG_GAME

Reply:
You're right
This is what it looks like now.

![image](https://user-images.githubusercontent.com/53777086/141058573-0cad5377-09e1-417b-9dbe-1603209eb5b1.png)